packages/esm-patient-vitals-app/src/vitals-biometrics-form/vitals-biometrics-form.utils.ts

```diff
@@ -1,7 +1,19 @@
 import isNumber from 'lodash/isNumber';
 import { type ConceptMetadata } from '../common';
 
-export function calculateBodyMassIndex(weight: number, height: number): number {
+export function calculateBodyMassIndex(weight: number, height: number, weightUnit: string, heightUnit: string): number {
```

Comment:
```suggestion
export function calculateBodyMassIndex(weight: number, height: number, weightUnit: 'lb' | 'lbs' | 'g', heightUnit: 'm' | 'cm' | 'in'): number {
```

Reply:
Not sure if this is the best way because then we have to call the function like this

```
const computedBodyMassIndex = calculateBodyMassIndex(
        weight,
        height,
        conceptUnits.get(config.concepts.weightUuid) as 'lb' | 'lbs' | 'g',
        conceptUnits.get(config.concepts.heightUuid) as 'm' | 'cm' | 'in',
      );
```